test/e2e/fixtures.go

```diff
@@ -474,3 +475,32 @@ func createTestBusyboxPods(tb testing.TB, data *TestData, num int, ns string, no
 
 	return podNames, podIPs, cleanupFn
 }
+
+// testServiceFromNode is a helper function which uses polling mechanism
+// to make sure that Service has been realized.
+func testServiceFromNode(t *testing.T, url, nodeName string, timeout time.Duration, statusReady bool) error {
+	cmd := fmt.Sprintf("curl -s --connect-timeout 1 --retry-connrefused %s", url)
+	err := wait.Poll(1*time.Second, timeout, func() (bool, error) {
+		t.Logf(cmd)
```

Comment:
The log here is meaningless. Perhaps we only need to log the cmd when error occurs.

Reply:
done.